Целочисленная арифметика/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Целочисленная_арифметика
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите четырехзначное число");
+            var number = int.Parse(Console.ReadLine());
+
+            var units = number % 1000 % 100 % 10;
```

Comment:
Число единиц можно определить проще
```suggestion
            var units = number % 10;
```